nginx.conf.src

```diff
@@ -60,22 +70,19 @@ http {
     location =/developer {
       return 302 https://open.gsa.gov/api/dap/;
     }
-    
+
     location / {
-      root <%= ENV["APP_ROOT"] %>/public;
+      root /home/vcap/app/_site;
       index index.html index.htm Default.htm;
-      <% if File.exists?(File.join(ENV["APP_ROOT"], "nginx/conf/.enable_directory_index")) %>
+
       autoindex on;
-      <% end %>
-      <% if File.exists?(auth_file = File.join(ENV["APP_ROOT"], "nginx/conf/.htpasswd")) %>
```

Comment:
this may be in use by other instances of this repo.

though, i don't think this is used by the cloud.gov instance.

Reply:
I did a search in the repo and didn't see it referenced anywhere else in the code outside of the nginx.conf. So I think we're good here.